include/robot_localization/ros_filter.h

```diff
@@ -633,6 +633,10 @@ template<class T> class RosFilter
     //!
     bool publishTransform_;
 
+    //! @brief Whether we're publshing the the inverse transfrom, ie. base_link_frame-world_frame transform
```

Comment:
typo: `publshing` -> `publishing`

Reply:
Also, you're missing the arrow head in `base_link_frame-world_frame` -> `base_link_frame->world_frame`